### ISSUE TITLE
WebView loads ancient browser

### ISSUE THREAD

**Issue by Metallicow** [wxWidgets/Phoenix#290]
Is there any way to get a different engine on windows than Internet Explorer...

using this code...

```
webview.WebView.New(self)
```

and loading https://www.python.org/ gives this...

```
Notice: Your browser is ancient and Microsoft agrees. Upgrade to a different browser or install Google Chrome Frame to experience a better web.
```

Is there a simple way to make webview use Firefox, Chrome, Opera, etc...?

**Comment by swt2c** [comment]
Probably not without significant changes to wxWidgets itself.  wxWidgets would need to be updated to support a newer WebView backend.

**Comment by Metallicow** [comment]
so this would be an issue to report on wxWidgets repo then...?

**Comment by swt2c** [comment]
Yes.  I've got a PR open to support the newer WebKit2, but that's for wxGTK only.

**Comment by swt2c** [comment]
@Metallicow this can be closed now that the Edge backend is available in 4.1.1?

**Comment by Metallicow** [comment]
Thank you, Scott. I am away. Happy you got the issue resolved.  -Meta
